Makefile

```diff
@@ -1,3 +1,9 @@
+# Version number
+VERSION=$(shell cat "./VERSION" 2> /dev/null)
```

Comment:
for the version is it possible to reuse `./tools/image-tag`?

it's what we use in CI to detect the version when we build the docker images.

Reply:
Yes, it's possible. Changed!